plugin-base/src/main/java/io/everytrade/server/plugin/impl/everytrade/parser/exchange/AnycoinExchangeSpecificParserV1.java

```diff
@@ -0,0 +1,269 @@
+package io.everytrade.server.plugin.impl.everytrade.parser.exchange;
+
+import io.everytrade.server.model.TransactionType;
+import io.everytrade.server.plugin.impl.everytrade.parser.exchange.anycoin.AnycoinBeanV1;
+import io.everytrade.server.plugin.impl.everytrade.parser.exchange.anycoin.AnycoinSortedGroupV1;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import static io.everytrade.server.plugin.impl.everytrade.parser.exchange.anycoin.AnycoinOperationTypeV1.OPERATION_TYPE_TRADE_REFUND;
+import static io.everytrade.server.plugin.impl.everytrade.parser.exchange.anycoin.AnycoinSupportedOperations.UNSUPPORTED_OPERATION_TYPES;
+import static java.util.stream.Collectors.groupingBy;
+
+public class AnycoinExchangeSpecificParserV1 extends DefaultUnivocityExchangeSpecificParser implements IExchangeSpecificParser,
+        IMultiExchangeSpecificParser<AnycoinBeanV1> {
+
+    List<AnycoinBeanV1> originalRows;
+    List<AnycoinBeanV1> unSupportedRows = new ArrayList<>();
+    List<AnycoinBeanV1> rowsWithOneRowTransactionType = new ArrayList<>();
+    List<AnycoinBeanV1> rowsWithMultipleRowTransactionType = new ArrayList<>();
+
+    LinkedList<AnycoinBeanV1> currenciesToBeIgnored = new LinkedList<>();
+    LinkedList<AnycoinBeanV1> notUsing = new LinkedList<>();
+
+    public AnycoinExchangeSpecificParserV1(Class<? extends ExchangeBean> exchangeBean, String delimiter) {
+        super(exchangeBean, delimiter);
+    }
+
+    @Override
+    public List<? extends ExchangeBean> convertMultipleRowsToTransactions(List<AnycoinBeanV1> rows) {
+        List<AnycoinBeanV1> result;
+        this.originalRows = rows;
+        filterRowsByType(rows);
+        List<AnycoinBeanV1> beans = prepareBeansForTransactions(rowsWithMultipleRowTransactionType, rowsWithOneRowTransactionType);
+        result = beans;
+        unSupportedRows.forEach(r -> r.setRowNumber(r.getRowId()));
+        result.addAll(unSupportedRows);
+        return beans;
+    }
+
+    private void filterRowsByType(List<AnycoinBeanV1> rows) {
+        rows
+                .forEach(r -> {
+                    if (r.getOperationType() == null || r.getType() == null) {
+                        unSupportedRows.add(r);
+                    } else if (r.getCoin().code().equalsIgnoreCase("ETH") &&
```

Comment:
The code should work for all .S currencies 
Avoid using strings

Reply:
fixed